form.schema.json

```diff
@@ -0,0 +1,217 @@
+              "description": "The label of the section"
+            },
+            "questions": {
+              "type": "array",
+              "description": "An array of all the questions in a section. Questions make up the fields of the form",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "Unique identifier of the question. Should be unique across all questions in the form. Prefer using camelCase for this property."
+                },
+                "label": {
+                  "type": "string",
+                  "description": "The question label that is displayed to the user"
+                },
+                "required": {
+                  "type": "boolean",
+                  "description": "Determines whether the question is a required field or not. Required fields must be answered before the form can be submitted. They may also get marked with a red asterisk (*) in the UI to denote them as required fields.",
+                  "default": false
+                },
+                "questionOptions": {
+                  "type": "object",
+                  "description": "Specifies metadata relating the question. These metadata are used to determine how the question is rendered in the UI, and how it is validated among other things.",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "Determines the datatype that the question accepts. The value of this property should be one of the supported datatypes."
+                    },
+                    "rendering": {
+                      "type": "string",
+                      "description": "Determines how the question is rendered in the UI. The value of this property should be one of the supported field types."
+                    },
+                    "concept": {
+                      "type": "string",
+                      "description": "The concept UUID of the backing concept for the question. This value should be a valid UUID of a concept that exists in the concept dictionary of your distribution."
+                    },
+                    "answers": {
+                      "type": "array",
+                      "description": "An array of all the possible answers for the question.",
+                      "properties": {
+                        "concept": {
+                          "type": "string",
+                          "description": "The backing concept UUID of the answer"
+                        },
+                        "label": {
+                          "type": "string",
+                          "description": "The label of the answer. This is what is displayed to the user"
+                        }
+                      },
+                      "required": ["concept", "label"]
+                    },
+                    "conceptMappings": {
```

Comment:
Is this right?  Why are there conceptMappings here?  What do these actually do?  I thought that the PR that Mark recently worked on got rid of this and made it so that any concept reference (eg. the "concept" of a question or an answer) could refer to either a uuid or a mapping.  I can't mention Mark here unfortunately, but can you weigh in @denniskigen  or @ibacher ?

Reply:
cc: @mogoodrich